packages/framework-core/src/services/graphql/query-helpers/graphql-query-filter-arguments-builder.ts

```diff
@@ -140,6 +140,8 @@ export class GraphqlQueryFilterArgumentsBuilder {
         in: { type: GraphQLList(GraphQLString) },
         beginsWith: { type: GraphQLString },
         contains: { type: GraphQLString },
+        regex: { type: GraphQLString },
+        iRegex: { type: GraphQLString },
```

Comment:
what does `iRegex` mean?

Reply:
it's like `ILIKE` in SQL. In this case is case insensitive regex